examples/step-12/doc/results.dox

```diff
@@ -55,17 +48,6 @@ and after five adaptive refinement steps.
 <img src="https://www.dealii.org/images/steps/developer/step-12.sol-2.png" alt="">
 <img src="https://www.dealii.org/images/steps/developer/step-12.sol-5.png" alt="">
```

Comment:
For now the solutions are identical. We can replace them in a follow-up?

Are you saying I should copy them for step-12b*?

Reply:
No. The documentation of older versions of deal.II still links to these pictures, so they need to stay in place. What I meant is that you now output the solution as VTK and it will actually look pretty. But you can't just replace the same files (with these names) because of the existing links to them. You need to call the new files differently (e.g., step-12.sol-5.9.2.png or some such).